azkaban-web-server/src/web/js/azkaban/test/test.js

```diff
@@ -7,8 +7,11 @@ function addClass(el, newClass) {
 var rewire = require('rewire'),
     dateJs = rewire('../util/date.js');
 
-var chai = require('chai');
-var assert = chai.assert;
+var moment = require('moment-timezone');
```

Comment:
Yes. We should do it when we have large extensively tests. For now, we may keep them together.